ci/kind/kind-setup.sh

```diff
@@ -300,6 +300,25 @@ function configure_vlan_subnets {
       docker_run_with_host_net iptables -t filter -A FORWARD -i ${vlan_interfaces[j]} -o ${vlan_interfaces[i]} -j ACCEPT
     done
   done
+
+  # Adding ipset rules to prevent SNAT based on flexibleIPAM e2e tests
+  if [[ $FLEXIBLE_IPAM == true ]]; then
+    docker_run_with_host_net iptables -t nat -D POSTROUTING ! -o $bridge_interface -s 192.168.240.0/24 -j MASQUERADE
+    
+    docker_run_with_host_net ipset create excluded_subnets hash:net
+    docker_run_with_host_net ipset add excluded_subnets 192.168.241.0/24
+    docker_run_with_host_net ipset add excluded_subnets 192.168.242.0/24
+    docker_run_with_host_net ipset list excluded_subnets
+    docker_run_with_host_net iptables -t nat -A POSTROUTING  ! -o  $bridge_interface -s 192.168.240.0/24 -m set ! --match-set excluded_subnets dst -j MASQUERADE
+
+    docker_run_with_host_net ipset create excluded_ipam_subnets hash:net
+    docker_run_with_host_net ipset add excluded_ipam_subnets 192.168.241.0/24
+    docker_run_with_host_net ipset add excluded_ipam_subnets 192.168.242.0/24
+    docker_run_with_host_net ipset add excluded_ipam_subnets 192.168.240.0/24
```

Comment:
could we use a single ipset? I don't think there is any harm in including `192.168.240.0/24` in `excluded_subnets`, since if the source and destination are both in `192.168.240.0/24`, the packet will be bridged.